src/parse.rs

```diff
@@ -289,4 +310,26 @@ mod tests {
         assert_eq!(fetches[0].message, 37);
         assert_eq!(fetches[0].uid, Some(74));
     }
+
+    #[test]
+    fn parse_search_ids_test() {
+        let lines = "* SEARCH 1600 1698 1739 1781 1795 1885 1891 1892 1893 1898 1899 1901 1911 1926 1932 1933 1993 1994 2007 2032 2033 2041 2053 2062 2063 2065 2066 2072 2078 2079 2082 2084 2095 2100 2101 2102 2103 2104 2107 2116 2120 2135 2138 2154 2163 2168 2172 2189 2193 2198 2199 2205 2212 2213 2221 2227 2267 2275 2276 2295 2300 2328 2330 2332 2333 2334 2335 2336 2337 2338 2339 2341 2342 2347 2349 2350 2358 2359 2362 2369 2371 2372 2373 2374 2375 2376 2377 2378 2379 2380 2381 2382 2383 2384 2385 2386 2390 2392 2397 2400 2401 2403 2405 2409 2411 2414 2417 2419 2420 2424 2426 2428 2439 2454 2456 2467 2468 2469 2490 2515 2519 2520 2521".as_bytes();
```

Comment:
The server response is usually split across multiple lines. May be worthwhile to include that behavior in the test too.

Reply:
Dovecot always responded with a single line in my tests but maybe I didn't hit the limit. I also didn't see any mention in the RFC. How would those multiple lines look? Are they multiple `* SEARCH` responses?

Like this?
```
* SEARCH 1 2 3 4
* SEARCH 5 6 7 8
```